PowerPointLabs/PowerPointLabs/SaveLab/SaveLabMain.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Windows.Forms;
+
+using Microsoft.Office.Interop.PowerPoint;
+
+namespace PowerPointLabs.SaveLab
+{
+    static class SaveLabMain
+    {
+        public static void SaveFile(Models.PowerPointPresentation currentPresentation)
+        {
+            // Opens up a new Save File Dialog
+            SaveFileDialog saveFileDialog = new SaveFileDialog();
+            Models.PowerPointPresentation newPresentation;
+
+            List<Models.PowerPointSlide> selectedSlides = currentPresentation.SelectedSlides;
+
+            // Setting for Save File Dialog
+            saveFileDialog.InitialDirectory = SaveLabSettings.GetSaveFolderPath();
+            saveFileDialog.Filter = "PowerPoint Presentations|*.pptx";
+            saveFileDialog.Title = "Save Selected Slides";
+            saveFileDialog.OverwritePrompt = true;
+
+            if (saveFileDialog.ShowDialog() == DialogResult.OK)
+            {
+                // Copy the Current Presentation under a new name
+                currentPresentation.Presentation.SaveCopyAs(saveFileDialog.FileName, PpSaveAsFileType.ppSaveAsDefault);
+
+                try
+                {
+                    // Re-open the save copy in the same directory in the background
+                    Presentations newPres = new Microsoft.Office.Interop.PowerPoint.Application().Presentations;
```

Comment:
Ultimately it would call base.Open(), which would still invoke the Presentations.Open() method, which would throw an error unless you install the VBA component. 

Note that the open methods you are referring to are in the main project, not in the test project. The VBA component is for the testing, and does not affect the main functionality. Using Open method in Test project is much more trickier, which needs the workaround of the VBA component.